pkg/datarepair/repairer/repairer.go

```diff
@@ -140,6 +140,12 @@ func (service *Service) process(ctx context.Context) (err error) {
 
 func (service *Service) worker(ctx context.Context, seg *pb.InjuredSegment) (err error) {
 	defer mon.Task()(&ctx)(&err)
+
+	insertedTime := seg.GetInsertedTime()
```

Comment:
What if an item in the queue doesn't have this time.

Reply:
Oh good point. Do we also need to worry about this?
```
seg *pb.InjuredSegment
var path string
err = tx.Tx.QueryRowContext(ctx, r.db.Rebind(`
	SELECT path, data FROM injuredsegments
	WHERE attempted IS NULL
	OR attempted < datetime('now','-1 hours')
	ORDER BY path LIMIT 1`)).Scan(&path, &seg)
```
If the `pb.InjuredSegment` struct has changed and the `data` stored in the `injuredsegments` table uses the old struct? I am not sure if it would still scan properly. Do you know?